_posts/projects/2015-10-4-epigraphing-19th-C.-novels.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: project
+title: "Epigraphing the 19th Century"
+tags:
+- Aaron Plasek
+category: theory-method
+type: experiment / archive
+published: true
+snippet: 123
+images:
+
+---
+
+#PROVOCATION
+
+From working with many 19th century novels, we know that the Bible and Shakespeare were frequently used as novel epigraphs, but little is known about who else was being quoted. Nor is anything known about the larger patterns of affiliation and citation assembled by these texts. How do epigraphs serve as "metadata" for different genres? And what can we infer about a particular text given the presence of a particular epigraph? Underlying these questions about the kinds of information the study of epigraph networks makes legible are questions of translation: how do the specific algorithmic affordances and limitations enacted through translation of physical books to XML pages serve to establish, stabilize, and challenge our historical and contemporary notions of credit, authority, and originality?
+
+Frequently ignored and occasionally made up, the epigraph is a textual genre defined both by its physical placement on the page and by the absence of the textual object being signposted. An epigraph attribution situates the text it prefaces within a larger constellation of texts and authors, and in this manner has an indexical function rather similar to scanning the spines of books on a shelf, flipping through a card catalog, or examining a record in a digital relational database. While in no way accepting that the affordances offered through the iterative construction of citation networks is ever a substitution for a critical method, a comparative approach to the different kinds of citations practices made visible by different networks of attribution provide an opportunity to reconsider how shared concepts that constitute a (disciplinary) field are produced, transmitted, and inhibited through our media-centric notions of consensus.   
+
+
+#WHAT WE'VE DONE
+
+Collecting nearly 3,000 XML-encoded 19th-century US novels from the Early American Fiction archive and [Wright American Fiction](https://github.com/iulibdcs/tei_text) archive, Aaron (in [collaboration with others](https://github.com/aaronplasek/19th_C._novel_scraper/graphs/contributors)) wrote a [scraper in python](https://github.com/aaronplasek/19th_C._novel_scraper) to collect the following epigraph and bibliographic information: 
+(1) novel author,
+(2) novel title,
+(3) novel publication date,
+(4) novel publication location,
+(5) epigraph text,
+(6) epigraph attribution,
+(7) author birth and death years, and 
+(8) file provenance. 
+This information is useful both for exploring epigraph-usage and for examining broader questions about the circulation and printing of novels in the 19th century, and could be quite easily adapted to digital techniques like mapping. **So far we've collected 7,689 epigraphs (as of 11 Oct 2015).**
```

Comment:
Move the updates to the top under ## updates heading (I'll talk about this today)